packages/edit-post/src/components/layout/index.js

```diff
@@ -123,6 +127,8 @@ function Layout( { styles } ) {
 			showBlockBreadcrumbs: select( editPostStore ).isFeatureActive(
 				'showBlockBreadcrumbs'
 			),
+			// translators: Default label for the Document in the Block Breadcrumb.
+			documentLabel: postTypeLabel || __( 'Document' ),
```

Comment:
I guess you could use `_x` here.

Reply:
Good call. Added to be more explicit about the word class. 👍